cmd/erasure-sets.go

```diff
@@ -267,13 +270,22 @@ func (s *erasureSets) monitorAndConnectEndpoints(ctx context.Context, monitorInt
 	// Pre-emptively connect the disks if possible.
 	s.connectDisks()
 
+	monitor := time.NewTimer(monitorInterval)
```

Comment:
Might as well be a `time.Ticker`, doesn't need resets.

Reply:
ticker cannot be used need to wait for connectDisks() to finish one full round @klauspost